common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -521,6 +521,8 @@ function (HTML5Video, Resizer) {
         if (this.videoPlayer.isPlaying()) {
             this.videoPlayer.runTimer();
         }
+        // Update the the current time when user seek. (YoutubePlayer)
+        this.videoPlayer.currentTime = time;
```

Comment:
I tried to understand the logic of that if/else but i didn't find any solid reason to remove that and have only line#525.